core/src/main/java/org/infinispan/topology/PersistentUUID.java

```diff
@@ -41,10 +40,12 @@ public static PersistentUUID fromString(String name) {
    }
 
 
+   @ProtoField(1)
```

Comment:
why not annotate `final UUID uuid`?

Reply:
I'm guessing it's because I converted this class before we even had the UUIDAdapter :see_no_evil: